src/cli/node/json_rpc_client.rs

```diff
@@ -115,6 +115,52 @@ pub fn get_pkh(addr: SocketAddr) -> Result<(), failure::Error> {
     Ok(())
 }
 
+#[allow(clippy::cast_possible_wrap)]
+pub fn get_utxo_info(addr: SocketAddr) -> Result<(), failure::Error> {
+    let mut stream = start_client(addr)?;
+    let request = r#"{"jsonrpc": "2.0","method": "getUtxoInfo", "id": "1"}"#;
+    let response = send_request(&mut stream, &request)?;
+    let utxo_info = parse_response::<UtxoInfo>(&response)?;
+
+    let utxos_len = utxo_info.utxos.len();
+    let mut table = Table::new();
+    table.set_format(*prettytable::format::consts::FORMAT_NO_BORDER_LINE_SEPARATOR);
+    table.set_titles(row![
+        "OutputPointer",
+        "Value (in wits)",
+        "Time lock",
+        "Ready for collateral"
+    ]);
+    for utxo_metadata in utxo_info.utxos.into_iter().sorted_by_key(|um| um.value) {
```

Comment:
Here you can sort by value and then by output pointer, so the list is sorted when there are many UTXOs with the same value:

```suggestion
    for utxo_metadata in utxo_info.utxos.into_iter().sorted_by_key(|um| (um.value, um.output_pointer)) {
```

But you need to implement `Ord` for `OutputPointer`, which should be something like:

```rust
impl Ord for OutputPointer {
    fn cmp(&self, other: &Self) -> Ordering {
        (self.transaction_id, self.output_index).cmp(&(other.transaction_id, &other.output_index))
    }
}
```